### DISCUSSION TITLE
README overhall

### DISCUSSION THREAD

**Discussion by clemens-fricke** [tataratat/gustaf#107] [Ideas]
Hey,

Let us add more images and less code (or at least show what the code actually produces) to the README. This is partly a visualization library and next to the logo we have no visualizations in the readme. And with over 100 lines, I think the "Quick Start" is a little bloated. Can we not just show a simple Image of a spline with a max of 10 lines of code where this is produced? 

In addition, the README is currently not self-coherent since it actually links to the optional dependencies link in the same file which does not exist anymore. I would really like to talk about how we can make the entry point into the Library simpler and more accessible.

Personally, I think a quick start is not a quick start if you have asserts in there making sure dimensions are actually correct to having multiple different show options.

/rantover

**Comment by j042** [comment]
sure, please feel free to create a PR
